tests/unit/schema/test.action.js

```diff
@@ -0,0 +1,62 @@
+import cloneDeep from 'lodash.clonedeep';
+
+import { validateAddon } from 'schema/validator';
+
+import { validManifest } from './helpers';
+import { assertHasMatchingError } from '../helpers';
+
+describe('/browser_action', () => {
+  it('should be allowed in manifest_version 2', () => {
+    let manifest = cloneDeep(validManifest);
```

Comment:
I am wondering if it would make sense to have a helper for that like (especially if `cloneDeep` is needed):

```
const manifest = makeValidManifest({
  manifest_version: 2,
  // and other overrides here
});
```

Reply:
I can definitely add one and use it in the new test added in this PR (but we may leave to a follow up changing all other existing places where we are using `cloneDeep(validManifest)`, I haven't looked but from what I recall I would expect most of the tests in tests/unit/schema may want to be updated to use the new helper once introduced).